src/Analyzers/CSharp/Tests/UseCompoundAssignment/UseCompoundCoalesceAssignmentTests.cs

```diff
@@ -35,7 +44,7 @@ public async Task TestBaseCase()
 @"class Program
 {
     private static string s_goo;
-    private static string Goo => s_goo [||]?? (s_goo = new string('c', 42));
+    private static string Goo => s_goo [|??|] (s_goo = new string('c', 42));
```

Comment:
switched to new test harness.  the new one is stricter about the test inputs, so a lot of existing tests are updated to be correct in that regard.